src-electron/db/query-command.js

```diff
@@ -1396,13 +1398,15 @@ async function selectAllCommandDetailsFromEnabledClusters(
     COMMAND.MUST_USE_TIMED_INVOKE,
     ENDPOINT_TYPE_CLUSTER.SIDE,
     CLUSTER.NAME AS CLUSTER_NAME,
-    ENDPOINT_TYPE_CLUSTER.ENABLED
+    CLUSTER.CLUSTER_ID,
+    ENDPOINT_TYPE_CLUSTER.ENABLED,
```

Comment:
Not a big issue but this can be removed from the diff as well

Reply:
CLUSTER.CLUSTER_ID is needed here as part of the return result.

Reply:
I meant the diff on `ENDPOINT_TYPE_CLUSTER.ENABLED,`. If you move it above `CLUSTER.CLUSTER_ID,` then the diff is cleaner but I see that we have an additional , . This is fine. I missed the ,